examples/Embeddings.jl

```diff
@@ -0,0 +1,17 @@
+using TensorBoardLogger #import the TensorBoardLogger package
+using Logging #import Logging package
+using TestImages
+using Flux, Flux.Data.MNIST
+
+logger = TBLogger("embeddinglogs", tb_append) #create tensorboard logger
+
+################log embeddings example: MNIST data################
+mat = MNIST.images()[1:100]
+mat = hcat(mat...)
+mat = reshape(mat, (28, 28, 100))
+mat = permutedims(mat, (3, 1, 2))
+mat = reshape(mat, 100, 28*28)
+metadata = MNIST.labels()[1:100]
+
```

Comment:
THis doesn't seems like a good thing to be embedding.
I suggest either:
 - Use word embeddings from Embeddings.jl
 - Use SVD or similar to find a low dimensional representation of MNIST.

Reply:
used SVD in the example